Bookish/Views/Book/BookQuery.cshtml

```diff
@@ -27,7 +27,11 @@
             <td>@Model.Author</td>
             <td>@Model.NumOfCopies</td>
             <td>@Model.NumOfAvailableCopies</td>
-            <td>@Model.Copies[0].Member.MemberId</td>
+            
```

Comment:
On line 14, I wouldn't use inline styles. Instead look at the classes offered by Bootstrap